src/model/user.js

```diff
@@ -3,6 +3,7 @@ import AppRequest from "component/request";
 var User = function(data) {
     this.id = data.id;
 
+    this.accountId = m.prop(data.account_id);
```

Comment:
Probably not, but if it doesn't change, then no harm. I just want to be consistent since it changes the way we access the data.